release_util_script_create_gpg.sh

```diff
@@ -23,7 +23,7 @@ gpg --batch --gen-key gen-key-script
 #gpg --quick-gen-key 'Sonny Martin <sonny.martin@diffblue.com>'
 #--------------------------------------------------------------------------------
 ## get key id
-GPG_KEYID=$(gpg -K | head -n4 | tail -n1 | tr -d ' ')
+GPG_KEYID=$( gpg --list-keys "Sonny Martin" | head -n1 | grep -Po "(?<=4096R/)[A-Z0-9]+")
```

Comment:
:grey_question: What is the difference between `-K` and `--list-keys`? The man page for gpg seems to suggest that `-K` lists all secret keys and `--list-keys` lists all public keys. I'm guessing the ID would be the same in both cases so it doesn't matter?
:question: I think I remember there being a problem with `grep -P` in models-library scripts on Macs, as Macs use a default version of grep that has strictly fewer features than GNU grep (but it's possible to install GNU grep using Homebrew). Should we test this script on a Mac, or just add a comment mentioning that there might be problems with this?

Reply:
>  I'm guessing the ID would be the same in both cases so it doesn't matter

Thanks for googling - so yes in this case public keys are sufficent, I find single letter flags a bit awkward to google so prefer to leave as I have it here. 

> `grep -P` on mac

Good question @sonodtt would you be able to check whether my replacement works on Mac?

Reply:
@sonodtt (also checking which version of grep you are using - it should be the default one on OS X, I forgot what it's called but I remember it not having "GNU" anywhere in its version name)